optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc

```diff
@@ -508,6 +508,10 @@ optaplanner.solver-config-xml::
 A classpath resource to read the solver configuration XML.
 Defaults to `solverConfig.xml`.
 If this property isn't specified, that file is optional.
+optaplanner.constraints-drl::
+A classpath resource to read the solver score DRL.
```

Comment:
Very soft suggestion: I can't explain why, but "solver score DRL" doesn't sound right to me, maybe because it implies there is a difference between "score DRL" and "solver score DRL"? (which there is not). 3 lines, above, "Solver configuration XML" is right. 
```suggestion
A classpath resource to read the score DRL.
```